### PR TITLE
Feature/users fixes [fix create user]

### PR DESCRIPTION
**Pull request by ah584d**

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
```
Comment (dimkinv) [inline .vscode/launch.json]: I recommend not to push the entire .vscode folder into git.
It contains local configurations which can be different from user to user. 

It's better to write it in the documentation instead and everyone who will want will add them manually.

File: src/config/environments/dev.ts [hunk 2]
```diff
@@ -4,6 +4,7 @@ import { Config } from './config.model';
 export const config: Config = {
     db: {
         connectionString: 'mongodb://admin:Aa123456@ds016128.mlab.com:16128/msw-dev',
+        // connectionString: 'mongodb://localhost',
```
Comment (dimkinv) [inline src/config/environments/dev.ts]: lets create local.ts file with this line

File: src/modules/persistence/users.persistence.service.ts [hunk 3]
```diff
@@ -25,21 +25,21 @@ export class UsersPersistenceService extends Logger implements IUsersPersistence
...
+            throw  [error, null];
         }
     }
 
     public async getById(id: string): Promise<UserDbModel> {
         try {
             const _dbId = new ObjectID(id);
             this.log(`UsersPersistenceService::getAll:: fetching user by id ${id}`);
-            return this.collection.findOne({ _dbId });
+            return await this.collection.findOne({ _id: _dbId });
```
Comment (dimkinv) [inline src/modules/persistence/users.persistence.service.ts]: if you will change the `_dbId` to `_id` this line can be simplified to ` return await this.collection.findOne({ _id });`

File: .vscode/launch.json (hunk 1 above)
Comment (ah584d) [reply .vscode/launch.json]: but it does not contains personal settings i debug with vscode, i must have it

File: src/config/environments/dev.ts (hunk 2 above)
Comment (ah584d) [reply src/config/environments/dev.ts]: ok

File: src/modules/persistence/users.persistence.service.ts (hunk 3 above)
Comment (ah584d) [reply src/modules/persistence/users.persistence.service.ts]: fixed

File: .vscode/launch.json (hunk 1 above)
Comment (dimkinv) [reply .vscode/launch.json]: it does, your debugging profile is personal, I use something else for example and by pushing .vscode folder you overriding my settings.
I suggest put it in .gitignore and then everyone will have their own settings, 

This folder also includes project settings, for example I've set that I dont see any of the `d.ts` files in the explorer, if I'd push it you won't see them either.

Comment (ah584d) [reply .vscode/launch.json]: ok you got it , i want to sleep ;-)

**Pull request merged**